src/main/java/internet/shop/model/User.java

```diff
@@ -20,6 +21,13 @@ public User(String name, String login, String password) {
         this.password = password;
     }
 
+    public User(String name, String login, String password, byte[] salt) {
+        this.name = name;
+        this.login = login;
+        this.password = password;
+        this.salt = salt;
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/50640794/94564678-ef574a00-0270-11eb-825e-9023a44087ce.png)
make the same with your constructors